app/config/Service.scala

```diff
@@ -32,15 +32,13 @@ final case class Service(host: String, port: String, protocol: String) {
 object Service {
 
   implicit lazy val configLoader: ConfigLoader[Service] = ConfigLoader {
-    config =>
-      prefix =>
+    config => prefix =>
+      val service  = Configuration(config).get[Configuration](prefix)
+      val host     = service.get[String]("host")
+      val port     = service.get[String]("port")
+      val protocol = service.get[String]("protocol")
 
-        val service  = Configuration(config).get[Configuration](prefix)
-        val host     = service.get[String]("host")
-        val port     = service.get[String]("port")
-        val protocol = service.get[String]("protocol")
-
-        Service(host, port, protocol)
+      Service(host, port, protocol)
   }
```

Comment:
why we are not using ServiceConfig?

Reply:
That's strange, I didn't remove that (as far as I'm aware). I'll add it back in now.

Reply:
Ah, it wasn't removed, it was just Scalafmt re-ordering things